src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,196 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node head;
+    private Node tail;
+    private int size;
+
+    private void validateIndexExclusive(int index) {
```

Comment:
[Private methods and classes should be after public ones in your class.](https://github.com/mate-academy/jv-linked-list/blob/master/checklist.md#private-methods-and-classes-should-be-after-public-ones-in-your-class)